src/core/services/google-maps-types.ts

```diff
@@ -547,13 +547,13 @@ export interface CalculatorResult {
 
 export type CalculateFunction = (marker: Marker[], count: number) => CalculatorResult;
 
-export interface MarkerClusterer {
+export interface IMarkerClusterer {
```

Comment:
That ist not that easy because tslint doesn't like it. The `js-marker-clusterer` package will create a global var called MarkerClusterer, which I have to declare `src/core/services/managers/cluster-manager.ts:12`